applications/difx2fits/src/sniffer.c

```diff
@@ -524,24 +502,11 @@ void deleteSniffer(Sniffer *S)
 {
 	if(S)
 	{
-		if(S->filebase)
-		{
-			free(S->filebase);
-		}
-		if(S->bp)
```

Comment:
I think these lines related to S->bp were actually here in 2.8.1? I.e. the block about dumping bandpasses didn't need to be deleted? Maybe there were multiple commits related to this and I'm missing something. I trust Walter with handling the sniffer in any case!